src/components/TimeControl/TimeControlPicker.tsx

```diff
@@ -194,7 +194,7 @@ export class TimeControlPicker extends React.PureComponent<
     update_period_time = (ev) => this.syncTimeControl({ period_time: parseInt(ev.target.value) });
     update_periods = (ev) =>
         this.syncTimeControl({
-            periods: Math.max(1, Math.min(300, parseInt(ev.target.value) || 1)),
```

Comment:
I hit merge on this too quick, this is still needed as folks can put in whatever they want in those input fields, so folks can create a game with 3248987 periods or whatever. (I should probably validate that on the server too.. but would be good to enforce that here as well)

Reply:
I will check again later. But as far as I remember the value comes from the numeric input with a Max and min correctly set. Users cannot change them as they want we just need to make sure we give it a sane initial value. Otherwise it just passes the value forward (like here) without changing it.

Reply:
They're set correctly there but input fields only use that for enforcing it when folks are using the up/down arrows or scroll wheel or whatever. If they type in a number, they can put in whatever they want and it'll accept it, which is the problem and the need for second check.

Reply:
Oh I see! I will propose something later tonight.

Reply:
@anoek what do you think about this approach: https://github.com/online-go/online-go.com/pull/1721 ?

Reply:
hmm there's still something missing here... a null value on the storage will still be sent to the backend even though the UI looks sane ...

Reply:
Yes, people can still create null byo-yomi games, presumably if it’s in local storage.